addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -178,7 +178,7 @@ def _generate_efaktur_invoice(self, delimiter):
             eTax['REFERENSI'] = number_ref
             eTax['KODE_DOKUMEN_PENDUKUNG'] = '0'
 
-            lines = move.line_ids.filtered(lambda x: x.move_id._is_downpayment() and x.price_unit < 0 and not x.display_type)
+            lines = move.line_ids.filtered(lambda x: x.move_id._is_downpayment() and x.price_unit < 0 and x.display_type not in ('line_note', 'line_section'))
```

Comment:
```suggestion
            lines = move.line_ids.filtered(lambda x: x.move_id._is_downpayment() and x.price_unit < 0 and x.display_type == 'product')
```

We only want the invoice lines here, not the payable/receivable and tax lines

Reply:
Ok, changed it!